lib/ansible/executor/task_executor.py

```diff
@@ -860,8 +860,9 @@ def _start_connection(self):
         python = sys.executable
 
         def find_file_in_path(filename):
-            # Check $PATH first, followed by same directory as sys.argv[0]
-            paths = os.environ['PATH'].split(os.pathsep) + [os.path.dirname(sys.argv[0])]
+            # Check $PATH for coverage dirs, followed by same directory as sys.argv[0]
```

Comment:
Could the path to `ansible-connection` be configurable, or is there something preventing that?